src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/PersonPicture.xaml

```diff
@@ -1,168 +1,84 @@
-                                IsTextScaleFactorEnabled="False" />
-
-                            <!-- Default font family to "Segoe MDL2 Assets" -->
-                            <FontIcon
-                                x:Name="BadgeGlyphIcon"
-                                x:DeferLoadStrategy="Lazy"
-                                AutomationProperties.AccessibilityView="Raw"
-                                Foreground="{ThemeResource PersonPictureEllipseBadgeForegroundThemeBrush}"
-                                FontFamily="{ThemeResource SymbolThemeFontFamily}"
-                                FontWeight="{TemplateBinding FontWeight}"
-                                VerticalAlignment="Center"
-                                HorizontalAlignment="Center" />
-                        </Grid>
-                    </Grid>
-                </ControlTemplate>
-            </Setter.Value>
-        </Setter>
-    </Style>
+<!-- MUX Reference PersonPicture.xaml, tag winui3/release/1.4.2 -->
+<ResourceDictionary xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:controls="using:Microsoft.UI.Xaml.Controls" xmlns:wasm="http://uno.ui/wasm" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:d="http://schemas.microsoft.com/expression/blend/2008" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" mc:Ignorable="d wasm">
+  <Style TargetType="controls:PersonPicture" BasedOn="{StaticResource DefaultPersonPictureStyle}" />
+  <Style x:Key="DefaultPersonPictureStyle" TargetType="controls:PersonPicture">
+    <Setter Property="Foreground" Value="{ThemeResource PersonPictureForegroundThemeBrush}" />
+    <Setter Property="Width" Value="96" />
+    <Setter Property="Height" Value="96" />
+    <Setter Property="FontFamily" Value="{ThemeResource ContentControlThemeFontFamily}" />
+    <Setter Property="FontWeight" Value="SemiBold" />
+    <Setter Property="IsTabStop" Value="False" />
+    <Setter Property="AutomationProperties.AccessibilityView" Value="Raw" />
+    <Setter Property="Template">
+      <Setter.Value>
+        <ControlTemplate TargetType="controls:PersonPicture">
+          <Grid x:Name="RootGrid">
+            <VisualStateManager.VisualStateGroups>
+              <VisualStateGroup x:Name="CommonStates">
+                <!-- Visual State when a Photo is available for display -->
+                <VisualState x:Name="Photo">
+                  <VisualState.Setters>
+                    <Setter Target="PersonPictureEllipse.Fill" Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.ActualImageBrush}" />
+                    <!-- UNO WORKAROUND: ImageBrush is not stretching properly https://github.com/unoplatform/uno/issues/4951 -->
+                    <wasm:Setter Target="PersonPictureImage.Source" Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.ActualImageBrush.ImageSource}" />
```

Comment:
This wasm-specific workaround might no longer be needed. Consider removing it and see if everything is working as expected.

Reply:
@Youssef1313 Indeed seems to be fixed, I haven't noticed that the second ongoing referenced PR is not for wasm. Thanks!